README.md

```diff
@@ -1,12 +1,4 @@
 
```

Comment:
![image](https://github.com/ernestkorir/vet_clinic/assets/93039359/8875871f-fbb3-4fd9-84b7-50691d25b265)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request professional and descriptive 😊